pkg/service/v1alpha1/node.go

```diff
@@ -258,6 +258,12 @@ func (ns *node) NodeUnpublishVolume(
 
 	targetPath := req.GetTargetPath()
 	volumeID := req.GetVolumeId()
+	if len(volumeID) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "NodeUnpublishVolume Volume ID must be provided")
+	}
+	if len(targetPath) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "NodeUnpublishVolume Target Path must be provided")
+	}
 
```

Comment:
Above 2 variables are being verified in validateNodeUnpublishReq()

Reply:
aah, removed